fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/PredicateStatisticsCalculator.java

```diff
@@ -212,7 +212,6 @@ public Statistics visitBinaryPredicate(BinaryPredicateOperator predicate, Void c
             if (!checkNeedEvalEstimate(predicate)) {
                 return statistics;
             }
-            Preconditions.checkState(predicate.getChildren().size() == 2);
```

Comment:
Why remove those checks?

Reply:
When building `BinaryPredicateOperator`, the constructor has ensure its children size. Actually, we didn't always add these check when we operate an operator.